opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -611,6 +611,19 @@ void injectionGroup(const Opm::Schedule&     sched,
         }
         return;
     }
+    {
+        iGrp[nwgmax + IGroup::VoidageGroupIndex] = group.insert_index();
+        for (const auto& [phase, inj_prop] : group.injectionProperties()) {
+            if (inj_prop.cmode == Opm::Group::InjectionCMode::VREP) {
+                if (inj_prop.voidage_group.has_value()) {
```

Comment:
I guess you could fuse these two conditionals as
```C++
if (inj_prop.cmode == Opm::Group::InjectionCMode::VREP &&
    inj_prop.voidage_group.has_value()) {
    // ...
}
```
which would save one level of nesting.